packages/amplify-category-auth/resources/schemas/awsCognito/CognitoCLIInputs.schema.json

```diff
@@ -93,7 +93,7 @@
           }
         },
         "userpoolClientRefreshTokenValidity": {
-          "type": "number"
+          "type": ["string", "number"]
```

Comment:
Should it just be "number" or does it accept both string and number?

Reply:
it accepts both , in CFN its passed as number